src/validatePrTitle.js

```diff
@@ -52,30 +55,28 @@ module.exports = async function validatePrTitle(
   }
 
   if (!result.type) {
-    throw new Error(
-      `No release type found in pull request title "${prTitle}". Add a prefix to indicate what kind of release this pull request corresponds to. For reference, see https://www.conventionalcommits.org/\n\n${printAvailableTypes()}`
-    );
+    errorMessage = `No release type found in pull request title "${prTitle}". Add a prefix to indicate what kind of release this pull request corresponds to. For reference, see https://www.conventionalcommits.org/\n\n${printAvailableTypes()}`;
+    throw new Error(errorMessage);
```

Comment:
When we throw here, the `core.setOutput` statement below is never called. You could introduce a new function that makes sure we always do both:

```js
function raiseError(message) {
  core.setOutput('errorMessage', message);
  throw new Error(message);
}
```

… and use that for all validation errors here.

That way you also don't need the variable introduced above.

Reply:
Oh, good catch, a silly mistake on my part. Will refactor that later.